todo_list/urls.py

```diff
@@ -16,14 +16,14 @@
 """
 from django.contrib import admin
 from django.urls import path,include
-from todo_list_app import views
+from todo_list_app.views import index, login_user,logout_user, RegisterUserView, VerifyUserEmailView
 
 urlpatterns = [
     path('todo/', include('todo_list_app.urls')),
     path('admin/', admin.site.urls),
-    path('', views.index, name='index'),
-    path('auth/login/', views.login_user, name='login'),
-    path('auth/logout/', views.logout_user, name='logout'),
-    path('auth/register/', views.RegisterUserView.as_view(), name='register'),
-    path('auth/verify-email/<user_id>/', views.verify_user_email, name='verify-email'),
+    path('', index, name='index'),
+    path('auth/login/', login_user, name='login'),
```

Comment:
Remove `/` at the end.
```suggestion
    path('auth/login', login_user, name='login'),
```